app/presenters/two_factor_auth_code/piv_cac_authentication_presenter.rb

```diff
@@ -8,13 +8,41 @@ def header
     end
 
     def help_text
-      t('instructions.mfa.piv_cac.confirm_piv_cac_html')
+      if aal3_policy.aal3_required? && !aal3_policy.multiple_aal3_configurations?
+        t('instructions.mfa.piv_cac.confirm_piv_cac_only_html')
+      else
+        t('instructions.mfa.piv_cac.confirm_piv_cac_html')
+      end
     end
 
     def piv_cac_capture_text
       t('forms.piv_cac_mfa.submit')
     end
 
+    def link_text
+      if aal3_policy.aal3_required?
+        if aal3_policy.multiple_aal3_configurations?
+          t('two_factor_authentication.piv_cac_webauthn_available')
+        else
+          ''
```

Comment:
is there a reason to have a blank string like this, and not just let it be `nil`?